venom/python/AudioProcessor/include/PyAudioProcessorEditor.h

```diff
@@ -0,0 +1,33 @@
+#include "juce_audio_processors/juce_audio_processors.h"
+#include "juce_audio_basics/juce_audio_basics.h" // for juce::AudioBuffer
+#include "juce_audio_devices/juce_audio_devices.h"
+#include <pybind11/embed.h> // everything needed for embedding
+
+#include <utility>
+#define PYBIND11_DETAILED_ERROR_MESSAGES
+
+namespace py = pybind11;
+
+// This class links the python class back to the C++ class
+class PyAudioProcessorEditor : public juce::AudioProcessorEditor{
```

Comment:
Actually it should be deleted as this class and AudioProcessor are inheriting from not pure virtual classes, so each of them contains objects of suppertype - that is maintained by juce which potentialy will cause issues, the correct approach should be to cast python object to cpp using pybind cast() method, as this will not create additional objects

Reply:
Please elaborate. I don't think we can just cast a complex python object with cast() . We should talk about it